libadaptivity/load_balance/src/Mesh.cpp

```diff
@@ -713,6 +713,8 @@ samfloat_t Mesh::idealElementDensity(const unsigned eid)
 {
   assert(element_list[eid].get_flags()&ELM_VOLUME);
 
+  samfloat_t det;
```

Comment:
```suggestion
  samfloat_t det{};
```

A possible remedy for the issue below.